doc/users/whats_new/new_rcparams_added.rst

```diff
@@ -0,0 +1,14 @@
+New rcParams added
+----------------------------------------------------------------------
+
+* 'lines.markerfacecolor' allows users to set the default markerfacecolor for line plots
+* 'lines.markeredgecolor' allows users to set the default markeredgecolor for line plots
+The Default color is set to none, and these rcParams are only used if the line does not have any value specifed for these params.
```

Comment:
Please wrap at 80 here as well. This line looks like it should be part of the previous bullet point, so add a new line above it.